tests/unit/Util/ConfigurationTest.php

```diff
@@ -451,8 +451,8 @@ protected function assertConfigurationEquals(Configuration $expectedConfiguratio
         );
 
         $this->assertEquals(
-            $expectedConfiguration->getTestSuiteConfiguration(),
-            $actualConfiguration->getTestSuiteConfiguration()
+            $expectedConfiguration->getTestSuiteConfiguration()->tests(),
+            $actualConfiguration->getTestSuiteConfiguration()->tests()
```

Comment:
When we compare the entire `TestSuite` object, the test becomes fragile as the set of declared classes can have changed between tests.

See https://github.com/sebastianbergmann/phpunit/blob/6.5.0/src/Framework/TestSuite.php#L131.